nightmare_firmware/servos.cpp

```diff
@@ -65,13 +71,11 @@ void servo::move(int ang){
 	ang = limit(map(ang, -120, 120, 0, 1000),0,1000); //angle comes in 2 bytes, 0..1000 means -120..120
 	angle = ang;
 	servo_write(
-		SERVO_HEADER, 
-		id, 
-		SERVO_MOVE_TIME_WRITE, 
-		ang & 0xFF, 
-		ang >> 8, 
-		0x00, 0x00, 
-		~((byte)(id+0x08+(ang & 0xFF)+(ang >> 8)))
+		id,
+		SERVO_MOVE_TIME_WRITE,
+		ang & 0xFF,
+		ang >> 8,
+		0x00, 0x00
```

Comment:
Are you sure it stays the same?
checksum is everything passed into servo_write and added together, then inverted (using `~`)
old code the checksum was `~((byte)(id+0x08+(ang & 0xFF)+(ang >> 8)))`, while new code checksum is `~(SERVO_MOVE_TIME_WRITE + (and & 0xFF) + (ang>>8))` right? Aren't you missing `id + 0x08`?

Reply:
EDIT: Oooh now I see, SERVO_MOVE_TIME_WRITE "=" 1 + 7 so you get that 0x08 in the end, same for others, alright, marking this as resolved